assets/queries/k8s/default_service_account_in_use/query.rego

```diff
@@ -0,0 +1,40 @@
+package Cx
+
+CxPolicy [ result ] {
+    document := input.document[i]
+    
+    document.kind == "ServiceAccount"
+    
+    metadata := document.metadata
+    metadata.name == "default"
+    
+    object.get(metadata, "automountServiceAccountToken", "undefined") == "undefined"
```

Comment:
From what I saw in the documentation, *automountServiceAccountToken" is at the resource level, not metadata. Check the first example in this [section](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#use-the-default-service-account-to-access-the-api-server), but also try to find other examples, as this the only one I could find.

Reply:
https://docs.bridgecrew.io/docs/bc_k8s_38

Reply:
https://www.cyberark.com/resources/threat-research-blog/securing-kubernetes-clusters-by-eliminating-risky-permissions#prevent-service-account-token-automounting-on-pods, I found another example, I think it's correct to leave out of metadata